framerail/src/routes/about/+page.svelte

```diff
@@ -0,0 +1 @@
+<h1>UNTRANSLATED:TODO: info page</h1>
```

Comment:
Not relevant for this PR obviously but the JSONRPC method for server info is `info`.